nnvm/src/top/nn/upsampling.cc

```diff
@@ -48,10 +61,15 @@ inline bool UpsamplingLayout(const NodeAttrs& attrs,
 }
 
 NNVM_REGISTER_OP(upsampling)
-.describe(R"(Perform nearest neighbor upsampling to input array.
+.describe(R"(Perform upsampling to input array with nearest neighbour or bilinear interpolation.
 
 - **data**: Input is 4D array of shape (batch_size, channels, in_height, in_width).
```

Comment:
ok, then add a doc here saying an additional weight tensor is required for bilinear upsampling.

Reply:
Document updated.